openedx/features/marketplace/constants.py

```diff
@@ -1,36 +1,100 @@
 from django.utils.translation import ugettext as _
 
+DJANGO_COUNTRIES_KEY_INDEX = 0
+DJANGO_COUNTRIES_VALUE_INDEX = 1
+
+MARKETPLACE_PARAM_QUERY_KEY = 'query'
+MARKETPLACE_PARAM_COUNTRY_KEY = 'country'
+MARKETPLACE_PARAM_CITY_KEY = 'city'
+
+ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY = 'delivery-services'
+ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY = 'funding'
+ORGANIZATIONAL_PROBLEM_TYPE_HEALTH_CARE_SUPPLIES_KEY = 'healthcare-supplies'
+ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY = 'human-resources'
+ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY = 'mentorship'
+ORGANIZATIONAL_PROBLEM_TYPE_ONLINE_TRAINING_KEY = 'online-training'
+ORGANIZATIONAL_PROBLEM_TYPE_OTHER_KEY = 'other-problem'
+ORGANIZATIONAL_PROBLEM_TYPE_REMOTE_WORKING_TOOLS_KEY = 'remote-working-tools'
+
+ORGANIZATIONAL_PROBLEM_CHOICES = (
+    (ORGANIZATIONAL_PROBLEM_TYPE_HEALTH_CARE_SUPPLIES_KEY, 'Healthcare Supplies'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_REMOTE_WORKING_TOOLS_KEY, 'Remote Working Tools'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_ONLINE_TRAINING_KEY, 'Online Training'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY, 'Delivery Services'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY, 'Mentorship'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY, 'Funding'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY, 'Human Resources'),
+)
+
+ORGANIZATIONAL_PROBLEM_CHOICES_VIEW = (
+    (ORGANIZATIONAL_PROBLEM_TYPE_HEALTH_CARE_SUPPLIES_KEY, 'Healthcare Supplies'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_REMOTE_WORKING_TOOLS_KEY, 'Remote Working Tools'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_ONLINE_TRAINING_KEY, 'Online Training'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY, 'Delivery Services'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY, 'Mentorship'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY, 'Funding'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY, 'Human Resources'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_OTHER_KEY, 'Other'),
+)
+
+ORGANIZATIONAL_SECTOR_HEALTH_AND_WELL_BEING = 'health-and-well-being'
+ORGANIZATIONAL_SECTOR_EDUCATION = 'education'
+ORGANIZATIONAL_SECTOR_GENDER_EQUALITY = 'gender-equality'
+ORGANIZATIONAL_SECTOR_SANITATION = 'sanitation'
+ORGANIZATIONAL_SECTOR_CLIMATE_CHANGES = 'climate-changes'
+ORGANIZATIONAL_SECTOR_CLEAN_ENERGY = 'clean-energy'
+ORGANIZATIONAL_SECTOR_ENVIRONMENTAL_CONSERVATION = 'environmental-conservation'
+ORGANIZATIONAL_SECTOR_WORK_AND_ECONOMIC_GROWTH = 'work-and-economic-growth'
+ORGANIZATIONAL_SECTOR_HUMAN_RIGHTS = 'human-rights'
+ORGANIZATIONAL_SECTOR_SOCIAL_JUSTICE = 'social-justice'
+ORGANIZATIONAL_SECTOR_ART_AND_CULTURE = 'art-and-culture'
```

Comment:
Please append `_KEY` to these as well, as you have done for `ORGANIZATIONAL_SECTOR_OTHER_KEY`, or come up with a naming scheme that's not as long and still gives meaning to the variables.